test/Hydration-test.js

```diff
@@ -77,19 +77,93 @@ describe('HydrationLog', function() {
     expect(userOneHydrationLogLog).to.be.an.instanceOf(HydrationLog);
   });
 
-  it.skip('should only have hydration stats for one user', function() {
-    expect(userOneHydrationLog.log.length).to.equal(9);
-    expect(userTwoHydrationLog.log.length).to.equal(2);
-  });
-
   it.skip('should have users hydration stats for all recorded dates', function() {
-    expect(userOneHydrationLog.log).to.have.all.keys('2019/06/15', '2019/06/16', '2019/06/17', '2019/06/18', '2019/06/19', '2019/06/20', '2019/06/21', '2019/06/22', '2019/06/23');
+    expect(userOneHydrationLog.log).to.have.deep.members([
+      {
```

Comment:
This isn't very dry, but I couldn't find an efficient way to test the entire contents of array of objects (as when log property was an object and we could test that it included _all_ keys) --- 
we could just test a few individual values, but that wouldn't *really* confirm the functionality of our constructor